### PR TITLE
switching over to browserify

### LINKED ISSUE #19: Browserify
We should migrate this to browserify, [post emitter](https://github.com/honeinc/post-emitter) is already up there. I think it should be relatively painless and would cleanup build steps/

### PR DESCRIPTION
**Pull request by jcblw**
let me know what you guys think

fixes #19

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<iframe width="600px" height="800px" src="http://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
-<script src="../build/embed.js" defer></script>
+<iframe width="600px" height="800px" src="https://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
```
Comment (cardoni) [inline examples/index.html]: Are we positive everything loads and renders perfectly over **http*s*://**?

I seem to recall some issues with fonts or something not being served correctly over ssl.  The wordpress site still having https-issues makes me think we need to double check our app too.

If you'd like, I'm happy to help out with this tomorrow.

Comment (jcblw) [inline examples/index.html]: broskes this is an example page, i was testing it using [srvdir](https://www.srvdir.net/) which is only https which is why this change.

Comment (cardoni) [inline examples/index.html]: You're not seeing stuff like this on your end?:
screenshot 2015-01-26 22 34 32

Comment (jcblw) [inline examples/index.html]: Yeah I do but that is just I assume because of the fonts they use for the quiz (not https) everything still logs correctly.
